cmd/pod-scaler/frontend/src/app/NotFound/NotFound.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { ExclamationTriangleIcon } from '@patternfly/react-icons';
+import {
+  PageSection,
+  Title,
+  Button,
+  EmptyState,
+  EmptyStateIcon,
+  EmptyStateBody,
+} from '@patternfly/react-core';
+import { useHistory } from 'react-router-dom';
+
+const NotFound: React.FunctionComponent = () => {
+  function GoHomeBtn() {
```

Comment:
Make this a separate component.

Reply:
This was taken from the React PatternFly Seed repo verbatim, will open a PR there to update!